frontend/src/Utilities/String/translate.js

```diff
@@ -1,18 +1,22 @@
 import createAjaxRequest from 'Utilities/createAjaxRequest';
 
 function getTranslations() {
-  return createAjaxRequest({
-    global: false,
+  let localization = null;
+  const ajaxOptions = {
+    async: false,
```

Comment:
`async: false` should be avoided, if we need to wait for the translations to load then we need to wait on a promise during initialization instead of forcing the browser to make the request synchonously.

Reply:
The promise will be needed. As doing this purely async with large number of translations can result in weird half translated scenarios in the UI (experienced on Radarr)

Reply:
ok, then I'll leave it as I have submitted?

Reply:
I'm going to change it a different way, I have a local change for it started so you can revert.

Reply:
done